### DISCUSSION TITLE
no such container (podman)

### DISCUSSION THREAD

**Discussion by KieranFleckney** [testcontainers/testcontainers-dotnet#797] [Q&A] [answered]
We are migrating from docker to podman however i am getting this issue when trying to run a test using test container. I can see the ryuk image but it will not run the container. Has anyone else had this issue.

Any help would be great

OS: Mac 2021 (M1)
Test Container: 2.1.0

```
Docker.DotNet.DockerContainerNotFoundException
Docker API responded with status code=NotFound, response={"cause":"no such container","message":"no container with name or ID \"cf8b2200fe32bddf2aef2221c10f0ee95f26368bba4093b0aa070d2cb8d61b0f\" found: no such container","response":404}
```

**Comment by HofmeisterAn** [comment, marked as answer]
As far as I know, Podman is not 100% compatible with the Moby API. You may have encountered this issue or a similar one: https://github.com/testcontainers/testcontainers-java/issues/5275. Please be aware that there are [known issues and limitations](https://github.com/search?q=repo%3Atestcontainers%2Ftestcontainers-java+Podman&type=issues) when using Podman with the Java implementation. I also assume that .NET may encounter similar issues due to inconsistencies in the Podman API.

**Reply by KieranFleckney** [reply]
hmm rather annoying, I thought it may work as from #694  seems to state they had it work just had include privileged flag set

**Reply by HofmeisterAn** [reply]
Yes, unfortunately. Something I just learned recently too. There are a few [issues](https://github.com/search?q=repo%3Acontainers%2Fpodman+Testcontainers&type=issues) in the Podman repository, including this one: https://github.com/containers/podman/issues/16400.

**Reply by KieranFleckney** [reply]
I guess there is no easy work around for now, not sure what my next step is now 😅 Thank you for your help
